app/presenters/return-logs/setup/meter-provided.presenter.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+/**
+ * Format data for the `/return-log/setup/{sessionId}/meter-provided` page
+ * @module MeterProvidedPresenter
+ */
+
+/**
+ * Format data for the `/return-log/setup/{sessionId}/meter-provided` page
+ *
+ * @param {module:SessionModel} session - The return log setup session instance
+ *
+ * @returns {object} page data needed by the view template
+ */
+function go(session) {
+  const {
+    id: sessionId,
+    data: { returnReference },
+    meterProvided
+  } = session
+
+  return {
+    backLink: `/system/return-logs/setup/${session.id}/units`,
```

Comment:
As you have the variable `sessionId` you could use that instead?
```suggestion
    backLink: `/system/return-logs/setup/${sessionId}/units`,
```